src/App.js

```diff
@@ -65,10 +64,7 @@ export default function App() {
     }
```

Comment:
This is short and sweet. I've seen some people using reduce for this but I don't see the point.